### ISSUE TITLE
two suggestions: class sorting and prompt improvement

### ISSUE THREAD

**Issue by fullstackneo** [windicss/windicss-intellisense#427]
Hello, I have some suggestions to improve plugin user experience, hope they will help.

1. **Class sorting**: when the plugin helps sort, it seems the customized class names will be put in the middle, I suggest to make them first place ranther than hide in windi classes, which might be not easy to locate a specific class. In addition, I suggest to put "flex" together with "justify-center items-center"  and "absolute" together with "left-3 top-2"
3. **Prompt**: let's say "bg-gray-200", when I simply delete "0" or "200", nothing will pop up. It would be a greater experience if the prompt would pop up rather than typing any num to make the prompt appear.

**Comment by jeremyjacob** [comment]
Agreed. Sorting unknown classes to the front would be a big improvement. Piggybacking off this to say that autocompletion working for user-defined shortcuts would be amazing.
